MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -167,7 +167,7 @@ public async Task SaveImage(BaseItem item, string url, ImageType type, int? imag
                 throw new HttpRequestException("Invalid image received.", null, response.StatusCode);
             }
 
-            var contentType = response.Content.Headers.ContentType.MediaType;
+            var contentType = response.Content.Headers?.ContentType;
```

Comment:
```suggestion
            var contentType = response.Content.Headers.ContentType?.MediaType;
```

Sorry, bad suggestion